modules/ROOT/pages/authentication-authorization/database-administration.adoc

```diff
@@ -179,7 +179,7 @@ Use `REVOKE` if you want to remove a privilege.
 ====
 
 Common errors, such as misspellings or attempts to revoke privileges that have not been granted or denied, will lead to notifications.
-Some of these notifications may be replaced with errors in a future major version of Neo4j.
+In Cypher 25, notifications for impossible `REVOKE` commands, where a user, a role, or a database does not exist, have been replaced with errors.
```

Comment:
do we have any examples of the Cypher 5 behaviour that we should now prepend with `CYPHER 5` and add notes on that they would fail in Cypher 25? Or is these 3 sentences the only thing on this topic that we had? 🤔

Reply:
I can not find anything more on this topic than those 3 sentences. Looks like all the examples we have are successful ones, not examples that will fail.